vncviewer/OptionsDialog.cxx

```diff
@@ -712,6 +721,38 @@ void OptionsDialog::createSecurityPage(int tx, int ty, int tw, int th)
   tx = orig_tx;
   ty += INNER_MARGIN;
 
+    /* Connection */
```

Comment:
This feature doesn't exist on Windows, so there needs to be an `#ifdef` around these things.